metricbeat/docs/modules_list.asciidoc

```diff
@@ -125,6 +125,8 @@ This file is generated! See scripts/docs_collector.py
 |<<metricbeat-metricset-nats-subscriptions,subscriptions>>   
 |<<metricbeat-module-nginx,Nginx>>     |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
 .1+| .1+|  |<<metricbeat-metricset-nginx-stubstatus,stubstatus>>   
+|<<metricbeat-module-oracle,Oracle>>  beta[]   |image:./images/icon-no.png[No prebuilt dashboards]    |  
```

Comment:
Aren't you going to include the dashboard you shared in a comment? I wouldn't include it as an "Overview" dashboard, as it still doesn't have some things I'd expect in a database dashboard, but it could be included as a "Tablespaces" dashboard.

Reply:
No no, it will go in a different PR